### ISSUE TITLE
in valid check,  can't handle selector like 'ul.classA.classB'

### ISSUE THREAD

**Issue by lyt9304** [rossPatton/stylint#301]
In our stylus code, there may be a selector like:`ul.classA.classB`.

It should be ok with valid check, because `ul` is a valid tag name.

But actually, stylint will report a warning, that is:

~~~
[Warning]: property is not valid
File: css/sem.styl
Line: 158: ul.list.cheliang-listing
~~~

I read the source code, then I found that 

~~~javascript
var stripRe = /(?=\S)\[\S+\]|(\.|#)(\w|-)+/ // /(?=\S)\[\S+\]/ // 2
~~~

stripRe only match one class name or one id name,  it cause problem when I use 2 or more class names or id names.

Maybe we can match more of conditions, like:

~~~javascript
var stripRe = /(?=\S)\[\S+\]|((\.|#)(\w|-)+)+/ // /(?=\S)\[\S+\]/ // 2
~~~

Thank you for your time to answer my question!

**Comment by rossPatton** [comment]
good idea, that's a very common use case. shouldn't be hard to accomodate
